blacklist/blacklist/Common/Utils/SegueIndentifier.swift

```diff
@@ -10,5 +10,6 @@ import Foundation
 
 enum SegueIndentifier: String {
     // Add your segues identifier here as enum, use .rawValue to use it
-    case SegueTest
+    case segueTest = "Test"
```

Comment:
You must not set a value to a string enum, the case is a string itself, please remove those values and try to use only the enum identifiers with the corresponding rawValue.

Reply:
How I have forgotten that. Thanks, men.